src/main/resources/seleniumGrid.properties

```diff
@@ -0,0 +1,4 @@
+gridHubURL=http://yourGridURL:gridPort/wd/hub
+#example: gridHubURL=http://192.168.10.10:4444/wd/hub
+browserval=firefox
```

Comment:
Is there any reason not to accept these values directly from properties passed to ATH proccess?